src/defaults/settings.js

```diff
@@ -10,6 +10,8 @@ export default //Renderer-specific settings
     stdy_col: 'STDY',
     endy_col: 'ENDY',
     seq_col: 'SEQ',
+    ongo_col: 'ONGO',
```

Comment:
Might want to make this optional ... We can assume there are no ongoing events unless it is provided.

Reply:
Will create a new issue for this under v1.0 for us to discuss.

Reply:
Created #41